core/src/main/java/io/questdb/PropServerConfiguration.java

```diff
@@ -1477,6 +1481,11 @@ public boolean enableTestFactories() {
             return false;
         }
 
+        @Override
+        public boolean getAllowTableRegistrySharedWrite() {
```

Comment:
wal-related properties should have `cairo` prefix

Reply:
no property in this file has `cairo` prefix